src/stylesheets/elements/_inputs.scss

```diff
@@ -112,7 +112,7 @@ select {
   appearance: none;
   background-color: $color-white;
   background-image: url('#{$image-path}/arrow-both.png');
-  background-image: url('#{$image-path}/arrow-both.svg');
+  background-image: none, url('#{$image-path}/arrow-both.svg'), url('#{$image-path}/arrow-both.png');
```

Comment:
Can you add this comment here on line 115 (above this rule): `// Ensure browsers that don't support SVG in background-image (IE 11 and below) fall back to PNG. See https://www.broken-links.com/2010/06/14/using-svg-in-backgrounds-with-png-fallback/.` ?

Reply:
@donjo minor thing: I don't know if it's documented but I think we try to do a 80 character limit for the code rule. Could probably drop "See" to another line.

Reply:
@maya good call, fixed!